transforms/remove-gridiron-import.js

```diff
@@ -2,6 +2,10 @@ module.exports = function (file, api, options) {
     var j = api.jscodeshift;
     var root = j(file.source);
 
+    if (!options.esprima) {
+        options.esprima = require("babel-core");
```

Comment:
Well all of the transforms should use `babel-core`, so yeah we need to figure out a way to apply options to all transforms.

Also your PR is failing on Travis.